qkeras/qconvolutional.py

```diff
@@ -474,10 +474,11 @@ def call(self, inputs, training=None):
 
   def get_config(self):
     config = super(QDepthwiseConv2D, self).get_config()
-    config.pop("filters", None)
-    config.pop("kernel_initializer", None)
-    config.pop("kernel_regularizer", None)
-    config.pop("kernel_constraint", None)
+    for x,y in (("filters", None),
```

Comment:
what is the benefit of this?

Reply:
Hi, There is not direct benefit when it comes to execution. However this is just a slight better way to doing the same. And makes the code a little more clean.